### DISCUSSION TITLE
Axure RP能不能拥有一席之地？

### DISCUSSION THREAD

**Discussion by lx1303011256** [ant-design/ant-design#41340] [General]
### What problem does this feature solve?

就是不喜欢IOS系统，就是不喜欢苹果，用不了sketch,为啥ant design要封锁AxureRP相关资源的下载。不用苹果，不用sketch就不配用ant design了么？ant design pro文档设计资源里提供了AxureRP资源的下载地址 http://library.ant.design/ ，可是访问之后直接跳转 https://ant-design.antgroup.com/docs/resources-cn ，满屏的sketch工具包，AxureRP用户就低你们一等？

### What does the proposed API look like?

开不开放无所谓了，毕竟是你们蚂蚁内部的东西，又不是服务广大开发者的，你们开心就好。

**Comment by afc163** [comment]
现在设计师都用 Sketch 了，Axure 资源文件维护不动无人更新了，给您带来不便非常抱歉。

**Comment by kgbchenlong** [comment]
AXURE已经没落了，大厂不用很正常。本来就是用的别人的开源贡献，还是多歇歇，少抱怨。
